docs/ORGANIZATION.md

```diff
@@ -1,110 +1,6 @@
-  * selectors
-    * CSS selector matching.
-  * servo
-    * Entry points for the servo application and libservo embedding library.
-  * style
-    * APIs for parsing CSS and interacting with stylesheets and styled elements.
-  * style_traits
-    * APIs to the style crate for crates that don't want to depend on the style crate for build speed reasons.
-  * util
-    * Assorted utility methods and types that are commonly used throughout the project.
-  * webdriver_server
-    * In-process server to allow manipulating browser instances via a WebDriver client.
-  * webgpu
-    * Implementation of threads for the WebGPU API.
-* etc
-  * Useful tools and scripts for developers.
-* mach
-  * A command-line tool to help with developer tasks.
-* ports
-  * winit
-    * Embedding implementation for the `winit` windowing library.
-* python
-  * servo
-    * Implementations of servo-specific mach commands.
-  * tidy
-    * Python package of code lints that are automatically run before merging changes.
-* resources
-  * Files used at run time. Need to be included somehow when distributing binary builds.
-* support
-  * android
-    * Libraries that require special handling for building for Android platforms
-  * rust-task_info
-    * Library for obtaining information about memory usage for a process
-* target
-  * debug
-    * Build artifacts generated by `./mach build --debug`.
-  * doc
-    * Documentation is generated here by the `rustdoc` tool when running `./mach doc`
-  * release
-    * Build artifacts generated by `./mach build --release`.
-* tests
-  * dromaeo
-    * Harness for automatically running the Dromaeo testsuite.
-  * html
-    * Manual tests and experiments.
-  * jquery
-    * Harness for automatically running the jQuery testsuite.
-  * power
-    * Tools for measurement of power consumption.
-  * unit
-    * Unit tests using rustc’s built-in test harness.
-  * wpt
-    * W3C web-platform-tests and csswg-tests along with tools to run them and expected failures.
+# Servo’s directory structure
 
-# Major dependencies
-* <https://github.com/servo/rust-mozjs>, <https://github.com/servo/mozjs>: bindings to SpiderMonkey
-* <https://github.com/hyperium/hyper>: an HTTP implementation
-* <https://github.com/servo/html5ever>: an HTML5 parser
-* <https://github.com/servo/ipc-channel>: an IPC implementation
-* <https://github.com/image-rs/image>: image decoders
-* <https://github.com/rust-windowing/winit>: cross-platform windowing and input
-* <https://github.com/jrmuizel/raqote>: a pure Rust 2D graphics library
-* <https://github.com/servo/rust-cssparser>: a CSS parser
-* <https://github.com/housleyjk/ws-rs>: a WebSocket protocol implementation
-* <https://github.com/servo/rust-url>: an implementation of the URL specification
-* <https://github.com/servo/webrender>: a GPU renderer
+Moved to the Servo book:
+
+- Servo’s architecture
+  - [Directory structure](https://book.servo.org/architecture/directory-structure.html)
```

Comment:
Nit: The architecture is different from the directory structure and a separate set of docs, right? Maybe this should just say "directory structure."

Reply:
“Servo’s architecture” is the section containing that chapter in the book’s sidebar, it’s just there to help you find the chapter (while being more stable than the chapter number)